### ISSUE TITLE
CC0008: Don't suggest for dynamic objects

### ISSUE THREAD

**Issue by pascalberger** [code-cracker/code-cracker#837]
# Bug

CC0008 currently suggests to use object initializer, even for dynamic objects.

``` csharp
dynamic email = new Email();
email.From = "foo@example.com";
email.To = "bar@example.com";
```

Current output after fix applied (if it is a code fix bug):

``` csharp
dynamic email = new Email()
{
    From = "foo@example.com",
    To = "bar@example.com"
}
```

Since this is a dynamic object, `From` and `To` might not be properties of the `Email` class and can therefore not be initialized with the object initializer syntax.

**Comment by giggio** [comment]
@pascalberger If `From` and `To` are not properties of the `Email` class the code will fail in the same way, will it not? The only difference is that the invocation is happening via dynamic dispatch in the first case, so CodeCracker will actually help you find an error.
The only scenario I can see this being a problem is with an `ExpandoObject`, or an object with similar patterns. So, in this scenario, this would raise an error where previously there was none, in a correct code. So, yes, this is a bug.

**Comment by giggio** [comment]
Closed in #860.
